src/amo/pages/UserProfileEdit/index.js

```diff
@@ -595,6 +604,24 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                 onChange={this.onFieldChange}
                 value={this.state.displayName}
               />
+              {isReviewer && user && user.reviewer_name !== undefined && (
+                <div>
+                  <label
+                    className="UserProfileEdit--label"
+                    htmlFor="reviewerName"
+                  >
+                    {i18n.gettext('Reviewer Name')}
+                  </label>
+                  <input
+                    className="UserProfileEdit-reviewerName"
+                    disabled={!user}
```

Comment:
I don't think this is possible, you can remove this attribute. The `input` won't be displayed without a user.

Reply:
978850dcd00fa09eb4c53da74330fcd373eaa5b1